Derpitcraft/README.md

```diff
@@ -34,21 +34,21 @@ All the following rules can be summarised simply, in one line. "Be kind to other
 
 3. **Pranking Policy**: Harmful pranks are not allowed. Harmless pranks are allowed and encouraged to create a fun experience for all players. Harmful pranking includes pranks that cause deaths, damage to items, and loss of items. Don't play pranks which won't be taken lightly. Try to judge this based on the player you are pranking. Try to take all kinds of harmless pranks in a light and fun manner. If you prank someone in a way that makes them die, or lose items, you are responsible for the items and need to return them within an hour. If you can't return them, then it will be counted as griefing. Always try to use common sense, and don't do pranks that can't be reversed. Think before pranking. If you don't want to be pranked at all, you can explicitly tell players to not prank you. If they don't respect your decision, report them, and we will take action. 
 
-4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the **dccoordinates** command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check #ds9-coordinates to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare/ lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Pt.5**). 
+4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the `/dccoordinates` command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check `#ds9-coordinates` to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare / lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Part 5**). 
```

Comment:
They are points. If you want to make them parts, then make them all headings and make these links. 

Link the channel if you can.

Reply:
will link